CorsixTH/Lua/world.lua

```diff
@@ -544,14 +554,14 @@ end
 
 function World:createEarthquake()
```

Comment:
I kill this function later in the pull request, so imo it's not worth documenting.